src/RaisedButton/RaisedButton.js

```diff
@@ -77,7 +77,7 @@ function getStyles(props, context, state) {
       position: 'relative',
       verticalAlign: 'middle',
       opacity: 1,
-      fontSize: '14px',
+      fontSize: raisedButton.fontSize || button.fontSize || '14px',
```

Comment:
I think that we should follow the `FlatButton` implementation. What about applying the fontSize at the root of the component and adding `raisedButton.fontSize: typography.fontStyleButtonFontSize` into `getMuiTheme.js`?